install_config/install/host_preparation.adoc

```diff
@@ -319,6 +319,32 @@ devices, which is not supported for production use and only appropriate for
 proof of concept environments. For production environments, you must create a
 thin pool logical volume and re-configure Docker to use that volume.
 
+Docker stores images and containers in a graph driver (a pluggable storage technology), such as *DeviceMapper*, 
+*OverlayFS*, and *Btrfs*. Each has advantages and disadvantages. For example, OverlayFS is faster than DeviceMapper 
+at starting and stopping containers, but is not Portable Operating System Interface for Unix (POSIX) compliant 
+because of the architectural limitations of a union file system, and does not yet support SELinux.
```

Comment:
OverlayFS does not support SELinux, is that true, according to https://github.com/openshift/openshift-docs/pull/8594/files#diff-f368171fd950e046a50929f2746793a4R334, it is said that overlay2 + selinux is supported on rhel7.4, the two statements conflict.